tensorbay/opendataset/UrbanObjectDetection/loader.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name, missing-module-docstring
+
+"""Dataloader of UrbanObjectDetection dataset."""
+
+import os
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import LabeledBox2D
+
+try:
+    import xmltodict
+except ModuleNotFoundError:
+    from tensorbay.opendataset._utility.mocker import xmltodict  # pylint:disable=ungrouped-imports
+
+_SEGMENT_NAMES = ("train", "val", "test", "trainval")
+DATASET_NAME = "UrbanObjectDetection"
+
+
+def UrbanObjectDetection(path: str) -> Dataset:
+    """`UrbanObjectDetection <http://www.rovit.ua.es/dataset/traffic/>`_ dataset.
+
+    The file structure should be like::
+
+        <path>
+            Annotations/
+                <image_name>.xml
+                ...
+            JPEGImages/
+                <image_name>.jpg
+                ...
+            ImageSets/
+                    train.txt
+                    val.txt
+                    test.txt
+                    trainval.txt
+                    ...
+                ...
+            ...
```

Comment:
What do these three `...` mean? You don't have to list them if you don't use them.